server/lib/nl/detection/utils.py

```diff
@@ -133,18 +134,52 @@ def empty_svs_score_dict():
   return {"SV": [], "CosineScore": [], "SV_to_Sentences": {}, "MultiSV": {}}
 
 
+def empty_var_candidates():
+  return dvars.VarCandidates(svs=[], scores=[], sv2sentences={})
+
+
+# Takes the detected svs and returns
+# 1. sv candidates: svs that are Statistical Variable or Topic
+# 2. non sv candidates: any other detected svs.
+def _get_sv_and_non_sv_candidates(
+    svs_scores_dict: Dict) -> tuple[dvars.VarCandidates, dvars.VarCandidates]:
+  sv_candidates = empty_var_candidates()
+  non_sv_candidates = empty_var_candidates()
+  sv_types = property_values(svs_scores_dict['SV'], 'typeOf')
+  for i, sv in enumerate(svs_scores_dict['SV']):
+    sv_type_list = sv_types.get(sv, [])
+    # a sv is considered a sv if any of its types are Statistical Variable or
+    # Topic.
+    is_sv = False
+    # We have some curated topics that are not a node in the kg, so assume topic
+    # if the sv starts with dc/topic
```

Comment:
Or check for type `Property` and classify it as non_sv?  So we don't add one more dep on `dc/topic/'

Reply:
I chose to check for SV or Topic instead of checking for Property because I want to try using "->" for linked properties so those would not return me a type ..

Reply:
hmm, ok, that feels brittle.  Have a comment?

Reply:
done